docker-compose.yml

```diff
@@ -21,6 +21,7 @@ services:
       - ./runtime/entrypoint.d:/st2-docker/entrypoint.d
       - ./runtime/st2.d:/st2-docker/st2.d
       - ./conf/stackstorm.env:/st2-docker/env
+    dns_search: .
```

Comment:
Since the change is not obvious, I think it makes sense to include a comment with a short description or maybe referencing this #74.

Reply:
Right. Included link to this PR.